### PR TITLE
updating tutorial mainly

### PR DESCRIPTION
**Pull request by eolivi**
Do not merge yet
This is not fully over.
It is just to show you were am I.
I'll finish soon.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -21,7 +32,7 @@ Using OpenMEEG
         * From a `Bash <https://raw.github.com/openmeeg/openmeeg_sample_data/master/compute_leadfields.sh>`_ script on Unix systems (Linux/Mac).
         * From a `BAT <https://raw.github.com/openmeeg/openmeeg_sample_data/master/compute_leadfields.bat>`_ file on Windows.
 
-`A PDF tutorial <http://openmeeg.gforge.inria.fr/download/tutorial-3.0.pdf>`_
+`A PDF tutorial <http://openmeeg.gforge.inria.fr/download/tutorial-2.4.pdf>`_
```
Comment (agramfort) [inline index.rst]: shouldn't we remove this ?

File: index.rst [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+  <div class="leadfield" style="float: right; margin: -7px 0 -10px 0;">
+
+.. image:: _static/leadfield.png
+   :width: 500px
+
+.. raw:: html
+
+  </div>
+
 What is OpenMEEG?
```
Comment (eolivi) [inline index.rst]: I have added this leadfield image, I think it is good to directly show what are the output and meshes of OpenMEEG

**Comment by eolivi** [comment]
mergeMe 
I think it is difficult for you to see the end result before the merge... or pull that branch locally + make html ?

Anyway, it removes a lot of dupplicate in the tutorial.rst + reformatting

File: index.rst (hunk 1 above)
Comment (eolivi) [inline index.rst]: not yet..

**Comment by eolivi** [comment]
once merged, we need to do the upload.sh

**Comment by agramfort** [comment]
nice

I pushed after a tiny cosmit

http://openmeeg.github.io/tutorial.html

**Pull request merged**